src/main/resources/application.yml

```diff
@@ -0,0 +1,35 @@
+server:
```

Comment:
Please implement the application.yml file without any depend on config-server 
Apply spring profile in order to control the config file base on the env
@nvthiet1995